docs/guide/browser.md

```diff
@@ -88,3 +102,7 @@ In this case, Vitest will run in headless mode using the Chrome browser.
 When using Vitest Browser, it's important to note that thread blocking dialogs like `alert` or `confirm` cannot be used natively. This is because they block the web page, which means Vitest cannot continue communicating with the page, causing the execution to hang.
 
 In such situations, Vitest provides default mocks with default returned values for these APIs. This ensures that if the user accidentally uses synchronous popup web APIs, the execution would not hang. However, it's still recommended for the user to mock these web APIs for better experience. Read more in [Mocking](/guide/mocking).
+
+### Mocking
+Mocking modules is not currently supported in the browser mode, but we are
```

Comment:
1. Don't add manual lines. 
2. I don't think we need to specify every feature we are working on in the documentation. The best way to add this note would be in the API section for `vi.mock`

Reply:
1- sure.
2- I believe having a place like a browser mode guide where the user sees many of the details is a good thing, it may be a big guide at some point, but I think it's worth it. Let me know your opinion.
At least that's how I expect the users to refer to the guide.

Reply:
No one who is using mocking will open this page instead of the API section. I've been fixing documentation for a year now, and I just know that this is not the place to describe the mocking behavior.

It will take a lot of time if you want to describe all features that are not implemented. If you know that something is not working in the browser mode, add a note in the API section.